src/main/java/org/spongepowered/api/command/spec/CommandSpec.java

```diff
@@ -163,6 +162,24 @@ public Builder child(CommandCallable child, String... aliases) {
         }
 
         /**
+         * Add a single child command to this command.
+         *
+         * @param child The child to add.
+         * @param aliases Aliases to make the child available under. First
+         *     one is primary and is the only one guaranteed to be listed in usage
+         *     outputs.
+         *
+         * @return this
+         */
+        public Builder child(CommandCallable child, List<String> aliases) {
```

Comment:
Actually you can just change the method to `child(CommandCallable child, Iterable<String> aliases)`, as it could still accept `List<String>`s and do not need extra casts below.